packages/airnode-node/src/evm/authorization/authorization-fetching.ts

```diff
@@ -114,15 +132,24 @@ export async function fetch(
   }
 
   // If there are no authorizer contracts then endpoint is public
-  if (isEmpty(fetchOptions.authorizers)) {
+  if (isEmpty(fetchOptions.authorizers.requesterEndpointAuthorizers)) {
     const authorizationByRequestIds = apiCalls.map((pendingApiCall) => ({
       [pendingApiCall.id]: true,
     }));
     return [[], Object.assign({}, ...authorizationByRequestIds) as AuthorizationByRequestId];
   }
 
+  // Skip fetching authorization statuses if found in config for a specific authorization type
+  // and requester address
+  const configAuthorizationsByRequestId = checkConfigAuthorizations(apiCalls, fetchOptions);
+
+  // Filter apiCalls for which a valid authorization was found in config
+  const apiCallsToFetchAuthorizationStatus = apiCalls.filter(
+    (apiCall) => !Object.keys(configAuthorizationsByRequestId).includes(apiCall.id)
```

Comment:
Nit: You can store `Object.keys(configAuthorizationsByRequestId)` in a variable before calling the `filter` function so it won't be called for each iteration (for each api call).